tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-packaging-pipeline.yml

```diff
@@ -14,16 +14,16 @@ jobs:
     - template: templates/set-version-number-variables-step.yml
     - template: templates/get-docker-image-steps.yml
       parameters:
-        Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_1_tensorrt7_2
+        Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_1_tensorrt8_0
```

Comment:
why cuda11_1, should be cuda11_4 per nvidia. (and we're using that for our CI pipeline anyway)